pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -191,14 +191,19 @@ func (c *Cluster) Start() error {
 	logger.Infof("wait timeout for healthy OSDs during upgrade or restart is %q", c.clusterInfo.OsdUpgradeTimeout)
 
 	// start the jobs to provision the OSD devices
-	logger.Infof("start provisioning the osds on pvcs, if needed")
+	logger.Infof("start provisioning the osds on PVCs, if needed")
 	c.startProvisioningOverPVCs(config)
 
+	if len(config.errorMessages) > 0 {
+		return errors.Errorf("%d failures encountered while running osds on PVCs in namespace %s: %+v",
```

Comment:
```suggestion
		return errors.Errorf("%d failures encountered while running osds on PVCs in namespace %q: %v",
```

Do not use `%+v`, otherwise the error.Wrap in the call above will add a stacktrace.

Reply:
got it. fixed.